lms/static/coffee/src/instructor_dashboard/data_download.coffee

```diff
@@ -107,17 +108,20 @@ class DataDownload
           @clear_display()
           @$download_display_text.html data['grading_config_summary']
 
-    @$calculate_grades_csv_btn.click (e) =>
+    @$async_report_btn.click (e) =>
       # Clear any CSS styling from the request-response areas
       #$(".msg-confirm").css({"display":"none"})
       #$(".msg-error").css({"display":"none"})
       @clear_display()
-      url = @$calculate_grades_csv_btn.data 'endpoint'
+      url = $(e.target).data 'endpoint'
       $.ajax
         dataType: 'json'
         url: url
-        error: (std_ajax_err) =>
-          @$reports_request_response_error.text gettext("Error generating grades. Please try again.")
+        error: std_ajax_err =>
+          if e.target.name == 'calculate-grades-csv'
+            @$grades_request_response_error.text gettext("Error generating grades. Please try again.")
+          else if e.target.name == 'ora2-response-btn'
+            @$grades_request_response_error.text gettext("Error getting ORA2 responses. Please try again.")
```

Comment:
Yes. Sorry, I copied the string but forgot to edit in my suggestion. I think all user facing strings should get a review from product so I haven't commented on any others.